examples/11-transactions/src/user.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Service } from 'typedi';
```

Comment:
@bedeho - all of the interesting stuff happens in this file.  To get transactions out of the box, you need to wrap your function with the `@Transaction` decorator, and then get an instance of a `TransactionManager` and pass this down to Warthog's BaseService (which will wrap everything in the transaction if requested.

Reply:
Also added tests in the test file below this.